src/Utilities/ErrorHandling/AbortWithErrorMessage.cpp

```diff
@@ -20,14 +20,13 @@ struct FillBacktrace {};
 // Convert the address to the virtual memory address inside the
 // library/executable. This is the address that addr2line and llvm-addr2line
 // expect.
-size_t convert_to_virtual_memory_address(const size_t addr) {
+size_t convert_to_virtual_memory_address(const void* addr) {
   Dl_info info;
   link_map* link_map = nullptr;
-  dladdr1(reinterpret_cast<void*>(addr),  // NOLINT
-          &info,
+  dladdr1(addr, &info,
           reinterpret_cast<void**>(&link_map),  // NOLINT
           RTLD_DL_LINKMAP);
-  return addr - link_map->l_addr;
+  return reinterpret_cast<size_t>(addr) - link_map->l_addr;
```

Comment:
Does it also work to return it as a pointer? It looks like the stream operator has a void* overload, but I'm not sure if it makes the formatting less useful for pasting into an addr2line command.

Reply:
That works at the tradeoff of an additional cast, but I think it's better with fewer stream formatting things because streams are painful :)